GNUmakefile

```diff
@@ -40,28 +32,27 @@ test-compile:
 	fi
 	@echo "==> Compiling test binary..."
 	go test -c $(TEST) $(TESTARGS)
-
-.PHONY: build test testacc vet fmt fmtcheck errcheck lint test-compile init-git-hooks
+.PHONY: test-compile
 
 # Build targets
 clean:
 	@echo "==> Cleaning build artifacts..."
-	rm -rf out
 	go clean ./...
+.PHONY: clean
```

Comment:
I think .PHONY should go above each target

Reply:
will do 👍